src/proto/transfer_gateway.proto

```diff
@@ -54,6 +54,15 @@ message TransferGatewayTokenWithdrawalSigned {
 
 // Contract Mapping
 
+message TransferGatewayAddAuthorizedContractMappingRequest {
```

Comment:
You don't need this, you can reuse `TransferGatewayAddContractMappingRequest`

Reply:
removed, but I can't see where we reuse message

Reply:
I meant you can just use `TransferGatewayAddContractMappingRequest ` instead of adding a new type, just set the contract address and not the other fields.